gammapy/makers/background/phase.py

```diff
@@ -19,32 +19,36 @@ class PhaseBackgroundMaker(Maker):
         on-phase defined by the two edges of each interval (edges are excluded)
     off_phase : `tuple` or list of tuples
         off-phase defined by the two edges of each interval (edges are excluded)
+    column_name : `str`
```

Comment:
Rename to the even more explicit `phase_column_name` or `column_name_phase`?

Reply:
Yes I guess it would be better.